subprojects/packagefiles/msgpackc-cxx/meson.build

```diff
@@ -1,27 +1,39 @@
 project(
-    'msgpackc-cxx', 'cpp',
-    version: '6.1.0',
-    meson_version: '>=0.42.0'
+  'msgpack-cxx', 'cpp',
+  version: '6.1.0',
+  meson_version: '>=0.47.0',
 )
 
 args = []
+deps = []
+
+boost_dep = dependency('boost', required: get_option('boost'))
+if boost_dep.found()
+  deps += [boost_dep]
```

Comment:
why not just pass this to dependencies directly?

Reply:
Ah, I would then only check do
```meson
if not boost_dep.found()
  args += ['-DMSGPACK_NO_BOOST']
endif
```
right?

Reply:
yeah. passing not found dependencies to dependencies: is fine. they won't be used/linked.